src/core/sprites/webgl/SpriteRenderer.js

```diff
@@ -241,9 +241,19 @@ export default class SpriteRenderer extends ObjectRenderer
         // copy textures..
         for (i = 0; i < MAX_TEXTURES; ++i)
         {
-            boundTextures[i] = rendererBoundTextures[i];
-            boundTextures[i]._virtalBoundId = i;
+            const bt = rendererBoundTextures[i];
+
+            if (bt._enabled === TICK)
+            {
+                boundTextures[i] = this.renderer.emptyTextures[i];
+                continue;
+            }
+
+            boundTextures[i] = bt;
+            bt._virtalBoundId = i;
+            bt._enabled = TICK;
         }
+        TICK++;
```

Comment:
Line 237  :  TICK++;
Line 256 :  TICK++;  

Is it Duplicated ?

Reply:
No, its not.

Reply:
I see. Thanks.